### ISSUE TITLE
Column default values

### ISSUE THREAD

**Issue by corentin-larose** [mysql-workbench-schema-exporter/mysql-workbench-schema-exporter#184]
Example table SQL:
```SQL
CREATE TABLE IF NOT EXISTS `accounts` (
[....]
  `created` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
  `updated` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP() ON UPDATE CURRENT_TIMESTAMP(),
[....]
  PRIMARY KEY (`id`))
ENGINE = InnoDB;
```
When generating entities, I get:

```php
 /**
     * @ORM\Column(type="datetime", options={"default": "CURRENT_TIMESTAMP"})
     */
    protected $created;

    /**
     * @ORM\Column(type="datetime", options={"default": "CURRENT_TIMESTAMP() ON UPDATE CURRENT_TIMESTAMP()"})
     */
    protected $updated = CURRENT_TIMESTAMP() ON UPDATE CURRENT_TIMESTAMP();
```

For a reason, the default value is copied on the second field.

**Comment by tohenk** [comment]
Fixed in 5c4110f9dd8bbd47d17fd1fdb8ed8687ddfc5bc5.

**Comment by corentin-larose** [comment]
Thanks, I also sent you an email to propose my help on some topics on this project.
